gems/pending/util/vcenter_thumb_print.rb

```diff
@@ -0,0 +1,25 @@
+require 'util/thumb_print'
+
+class VcenterThumbPrint < ThumbPrint
+  def initialize(host)
+    super(host)
+  end
```

Comment:
@chessbyte I believe I had some other code in there originally that I removed.  You are correct - I will remove it.